.github/workflows/build_test.yaml

```diff
@@ -121,6 +121,17 @@ jobs:
         run: |
           scripts/opentitan/ot-tidy.sh --ci -p build-clang
 
+  lint-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Don't shallow clone, we need to see all commits.
+      - name: Create local branch
+        run: git branch "${{ github.base_ref }}" "origin/${{ github.base_ref }}"
```

Comment:
Why creating a local branch rather than just use `origin/` in the script?

Reply:
If you want to make script generic you should add an argument and pass in `origin/${{ github.base_ref }}` instead.

Reply:
I want to run the script locally and my remotes are called `origin`, `upstream`, and `lowrisc` (for my fork, `qemu/qemu`, and `lowrisc/qemu`). I didn't want to hardcode `origin` in the script.

Reply:
In which case you should take the second suggestion and pass by arguments instead, rather than relying on GitHub environment variables.

Reply:
Thanks, done